packages/flutter/test/material/text_selection_theme_test.dart

```diff
@@ -58,6 +58,8 @@ void main() {
   });
 
   testWidgets('Empty textSelectionTheme will use defaults', (WidgetTester tester) async {
+    final Color primaryColor = Color(ThemeData.fallback().colorScheme.primary.value);
```

Comment:
I think this has to be the same as `Color(ColorScheme.light().primary.value)`. Only pointing that out because I wish we wouldn't use ThemeData.fallback(), since it needlessly obscures ThemeData.light().

Reply:
I just switched everything to color constants for the default values so this is no longer an issue.